### ISSUE TITLE
Event design and naming how-to

### ISSUE THREAD

**Issue by robrap** [openedx/openedx-events#76]
We should consider adding a how-to for adding events, that may pull from and reference ADRs, but would be a clear how-to document.

One potentially missing detail, is whether all Data objects should have the suffix "Data"? See https://github.com/openedx/openedx-events/blob/6e48994c9547466c7bff7fd1e78bb927c85fab68/openedx_events/learning/data.py#L142 for one example that doesn't match this convention. Separate and related, do we want linting to help enforce any conventions?

Do we want to document known subdomains? (I have additional questions about this, but better left to a separate issue.)

**Comment by mariajgrimaldi** [comment]
During the event's design, our primary focus was: event identification & the implemented name of the event, which we added in the [naming conventions ADR](https://github.com/openedx/openedx-events/blob/main/docs/decisions/0002-events-naming-and-versioning.rst), so the data classes were created more organically. Now, we should decide if we need to make that naming a convention.

> We should consider adding a how-to for adding events
Yes, I remember doing this for the conference. I'll brush up on what I wrote in a new PR.
